servers/rendering/renderer_rd/shaders/particles.glsl

```diff
@@ -67,7 +67,7 @@ struct FrameParams {
 	float delta;
 
 	uint frame;
-	uint pad0;
+	float amount_ratio;
```

Comment:
The same changes are needed in the Compatibility renderer. https://github.com/godotengine/godot/blob/master/drivers/gles3/shaders/particles.glsl